src/use-cases/saveEditorPage.ts

```diff
@@ -1,12 +1,12 @@
 import { logger } from '../logger';
-export const saveEditorPage = (submissions: Array<{ id: string; editorDetails: {} }>): ((_, { id, details }) => {}) => (
-    _,
-    { id, details = {} },
-): {} => {
+export const saveEditorPage = (
+    submissions: Array<{ id: string; lastStepVisited: string; editorDetails: {} }>,
+): ((_, { id, details }) => {}) => (_, { id, details = {} }): {} => {
     logger.info(`saveEditorPage(${id})`);
     const submissionIndex = submissions.findIndex(submission => submission.id === id);
     if (submissionIndex !== -1) {
         submissions[submissionIndex].editorDetails = details;
+        submissions[submissionIndex].lastStepVisited = 'editors';
```

Comment:
Is this how xpubs lastStepVisited is working? I think we're just storing the whole url path string not just the step (not saying this isn't a better way to do this but we'll need to be backwards compatible with data stored)

Reply:
This is just mimicking submission currently, i'll take a look now to see how its stored.